File: java/empleados/Comercial.java
public class Comercial extends Empleado{
    private double comision;

    public Comercial(String nombre,int edad,double comision){
        super(nombre,edad);
        if(edad > 30 && comision > 200){
            this.setSalario(comision + this.PLUS);
        }else{
            this.setSalario(comision);
        }
        this.comision = comision;
    }

    @Override
    public String toString(){
        return "\nNombre: " + this.getNombre() +
        "\nEdad: " + this.getEdad() + 
        "\nComision: " + this.comision + 
        "\nSalario: " + this.getSalario();
    }

}

File: php/Empleados/Comercial.php
<?php



class Comercial extends Empleados{
    private $comision;

    public function __construct($nombre,$edad,$salario,$comision){
        if($edad > 30 && $comision > 200){
            $salario = $salario + $comision + $this->PLUS;
        }else{
            $salario = $salario + $comision;
        }

        parent::__construct($nombre,$edad,$salario);
        $this->comision = $comision;
    }

    public function getEmpleado(){
        return parent::getEmpleado() . "\n Comision: " . $this->comision;
    }

}